core/s3.py

```diff
@@ -5,6 +5,10 @@
 """
 from flask import current_app, g
 import boto3
+import logging.config
+
+logging.config.fileConfig('logging.config')
+log = logging.getLogger()
```

Comment:
i herd there's this thing `current_app.logger`...

am i wrong on these?

Reply:
Nope. You are correct. Completely overlooked that on my part.